functions/tests/test_stripe_checkout.py

```
@@ -0,0 +1,727 @@
+
+        mock_db.collection.return_value.document.return_value.get.side_effect = [
+            mock_event_doc,
+            mock_organiser_doc,
+        ]
+
+        # Mock Stripe checkout session creation
+        mock_checkout = Mock()
+        mock_checkout.url = "https://checkout.stripe.com/session_123"
+        mock_stripe_create.return_value = mock_checkout
+
+        result = create_stripe_checkout_session_by_event_id(
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)